database/migrations/2023_05_12_103223_create_menu_items_table.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\DB;
+
+class CreateMenuItemsTable extends Migration
```

Comment:
All the DB::statment commands here should be done with Laravel's migration functions: https://laravel.com/docs/8.x/migrations#creating-tables